RFC0000-RFC-Process.md

```diff
@@ -35,49 +40,63 @@ All RFC documents shall follow the [RFC template](RFCNNNN-New-RFC-Template.md).
 
 ## RFC Workflow
 
-RFCs may go through the following stages:
+RFCs go through the following stages:
 
-### Draft
+### Draft PR
 
 This is the initial draft of an RFC posted for comments and considered a work-in-progress.
 
-* New proposed drafts should be submitted as a Pull Request from the Author's fork into the `Draft-Accepted` folder.
-* The Author shall ensure that the `Allow edits from maintainers` box is checked so that a Committee Member can assign the next RFC number.
-* If the Pull Request has followed the correct template and process, a Committee Member will assign the label `Review - Open for comments` to the Pull Request to indicate that anyone can comment on the content of the submission.
-Typically, one or two months is allowed for comments, though this may be extended if a submission is particularly contentious or hasn't received enough feedback for the Committee to feel comfortable making a decision.
-* When the Committee closes the comment period, the Author should update the RFC and Pull Request with a new commit to address the comments.
-* The Committee shall vote to merge or reject the RFC.
-Note: the Committee may be slower to respond to RFCs where the Author has indicated that they do not plan to implement the RFC.
-
-### Draft-Accepted
+* New drafts should not be submitted until the appropriate Working Group has decided that an RFC is
+  required in order for an experimental implementation to be accepted.
+* Drafts should be submitted as a draft Pull Request from the Author's fork into the
+  `Draft-Accepted` folder.
+* The Author shall ensure that the `Allow edits from maintainers` box is checked so that a Committee
+  Member can assign the next RFC number.
+* As a draft PR, contributors are encouraged to comment on the content of the submission.
 
-The PowerShell Committee has reviewed the RFC and comments, and has voted to accept the RFC as a Draft.
+### Non-Draft PR
 
-* New comments are not being sought.
-* No one has begun implementing the RFC, and there are no current plans to implement the RFC.
-In this case, the Committee will create `up-for-grabs` Issues in the [PowerShell](https://github.com/PowerShell/PowerShell) repository.
+* After two months, and when the Author feels that they've addressed feedback on the RFC,
+  the Author should move the RFC PR out of the draft state.
+  This signals to the Committee that the RFC is ready for their formal review.
+* At this point, the Committee will add a `Review - Committee` label.
 
-### Experimental
+### Experimental Feature
```

Comment:
Who and how strong will track timelines so that RFCs to go through all the steps on time?

Reply:
Unfortunately, I don't think any of this new process is going to magically ensure that features make it into a particular release. In fact, given the need for real-world usage data, we expect that some features will have to churn across multiple versions for us to have confidence that they should be turned on in non-preview releases by default (as you've seen with some of the PS6 experimental features that are still experimental).